repository/MonticelloFileTree-Git.package/MCFileTreeGitRepository.class/instance/basicStoreVersion..st

```diff
@@ -24,7 +24,7 @@ basicStoreVersion: aVersion
 					[ :n | stream nextPut: '--author=' , (n at: #name) , ' ' , (n at: #email) ].
 			stream
 				nextPut: '-m';
-				nextPut: (self escapeForShell: aVersion info message convertToSystemString);
+				nextPut: (self escapeForShell: aVersion info message asString convertToSystemString);
```

Comment:
#asString will convert anything to a string representation. For a Text instance, this is what we want. But for other objects, is this really what we expect, or is it a case of silencing type errors?